CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 Pkg v1.7 Release Notes
 ======================
 
+- Regisries downloaded from the Pkg Server (not git) are no longer uncompressed into files but instead read directly from the compressed tarball into memory. This improves performance on
```

Comment:
```suggestion
- Registries downloaded from the Pkg Server (not git) are no longer uncompressed into files but instead read directly from the compressed tarball into memory. This improves performance on
```

Reply:
```suggestion
- Registries downloaded from the Pkg Server (not git) are no longer uncompressed into files but instead read directly from the compressed tarball into memory. This improves performance on
```
I suggest using `--force-with-lease` instead of `--force` :slightly_smiling_face:

Reply:
Ouch...